### PR TITLE
#377 Label prop override

### PR DESCRIPTION
**Pull request by 3rdvision**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-robin-revamp/issues/377 |
| Decisions | - Allow label prop override for translations <br> - Pretty |

### CODE REVIEW

File: react/components/molecules/chat-message/chat-message.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ export class ChatMessage extends PureComponent {
             status: PropTypes.string,
             statusProps: PropTypes.object,
             replies: PropTypes.number,
+            repliesLabelSingular: PropTypes.string,
+            repliesLabelPlural: PropTypes.string,
```
Comment (joao-conde) [inline react/components/molecules/chat-message/chat-message.js:23]: replyLabel e repliesLabel talvez?

File: react/components/organisms/chat/chat.js [hunk 2]
```diff
@@ -22,9 +22,12 @@ export class Chat extends PureComponent {
                     username: PropTypes.string.isRequired,
                     date: PropTypes.number.isRequired,
                     message: PropTypes.string,
+                    text: PropTypes.string,
                     status: PropTypes.string,
                     statusProps: PropTypes.object,
                     replies: PropTypes.number,
+                    repliesLabelSingular: PropTypes.string,
+                    repliesLabelPlural: PropTypes.string,
```
Comment (joao-conde) [inline react/components/organisms/chat/chat.js:30]: Same as above

File: react/components/organisms/profile/profile.js [hunk 3]
```diff
@@ -12,6 +12,7 @@ export class Profile extends Component {
         return {
             account: PropTypes.object.isRequired,
             actions: PropTypes.object,
+            buttons: PropTypes.array,
```
Comment (joao-conde) [inline react/components/organisms/profile/profile.js:15]: Do we want to replace the entirety of the buttons? This is for localization later right? Why not, similar to the other components above, just have a shareContactText and signOutText?

**Review by joao-conde — CHANGES_REQUESTED** [review]

**Review by joao-conde — APPROVED** [review]

**Review by NFSS10 — CHANGES_REQUESTED** [review]
Revert the SVG change

imagem

**Comment by 3rdvision** [comment]
> Revert the SVG change
> 
> imagem

Pretty did this...

**Review by NFSS10 — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Pull request merged**